### ISSUE TITLE
About the baseline model eventx

### ISSUE THREAD

**Issue by YYYYWD** [RingBDStack/KPGNN#2]
Thank you for your work!
In the code of eventx you provided, I only saw the place to set the test set. I want to know if the eventx model does not need to set the training set? Just give the mask and index of the test set?

**Comment by songzuolong1997** [comment]
I have the same question too
